### ISSUE TITLE
Unable to run tests from the Contributing docs

### ISSUE THREAD

**Issue by javierjulio** [activeadmin/activeadmin#4961]
@deivid-rodriguez I've pulled in the latest master changes but following the instructions on the Contributing guidelines I've been unsuccessful. The `bundle install` portion completes fine but I have an issue with `bundle exec appraisal install`. First I notice an issue about databaseCleaner not being checked out and it suggests to run `bundle install` as mentioned that runs fine and completes successfully (even if re-run). This is the output I get after running `bundle exec appraisal install`:

```
Bundler could not find compatible versions for gem "railties":
  In snapshot (rails_42.gemfile.lock):
    railties (= 4.2.8.rc1)

  In Gemfile:
    activeadmin was resolved to 1.0.0, which depends on
      railties (< 5.2, >= 4.2)

    activeadmin was resolved to 1.0.0, which depends on
      coffee-rails was resolved to 4.2.1, which depends on
        railties (< 5.2.x, >= 4.0.0)

    activeadmin was resolved to 1.0.0, which depends on
      inherited_resources (~> 1.7) was resolved to 1.7.2, which depends on
        railties (< 5.2.x, >= 3.2)

    activeadmin was resolved to 1.0.0, which depends on
      jquery-rails was resolved to 4.3.1, which depends on
        railties (>= 4.2.0)

    rails (= 4.2.8) was resolved to 4.2.8, which depends on
      railties (= 4.2.8)

    activeadmin was resolved to 1.0.0, which depends on
      sass-rails was resolved to 5.0.6, which depends on
        railties (< 6, >= 4.0.0)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

**Comment by deivid-rodriguez** [comment]
Uggg one more reason to keep our lock files version controlled. It seems like last time you installed this we were using `4.2.8.rc1`...

```
BUNDLE_GEMFILE=gemfiles/rails_42.gemfile bundle update rails
```

should do the trick, I think.

**Comment by javierjulio** [comment]
Yes I remember that was a frustrating point for you. Hopefully we can do lock files if it helps. I ran that but I'm still getting the same output. 😞 I tried removing the lock file (gemfiles/rails_42.gemfile.lock) to see if I could just run `bundle exec appraisal install` successfully but that failed. I then tried to run `bundle install` but now that's failing half way through installing gems too. I'm not sure what I'm messing up here. Is there a way I can restart if you think that might help?

**Comment by deivid-rodriguez** [comment]
What about `BUNDLE_GEMFILE=gemfiles/rails_42.gemfile bundle update`?

Removing the lock file and doing `BUNDLE_GEMFILE=gemfiles/rails_42.gemfile bundle install` should work too I guess.

Which error are you getting? Can you post it?

**Comment by javierjulio** [comment]
@deivid-rodriguez all good now. Thanks so much for the help! The error was that there wasn't a Gemfile.lock in the root activeadmin directory which is weird because in Terminal it showed up. I realized it wasn't really there though since I opened Finder and learned its an alias. That was unexpected but makes that it is knowing the gem is tested against multiple gemfiles. I just had to delete that and the previous commands would now run successfully. I knew it was something simple I was just really stumped.
